FLEDGE.md

```diff
@@ -589,11 +593,11 @@ Buyers have three basic jobs in the on-device ad auction:
+Buyers may want to make on-device decisions that take into account real-time data (for example, the remaining budget of an ad campaign).  This need can be met using the interest group's `trustedBiddingSignalsURL`, `trustedBiddingSignalsKeys`, and, optionally, `trustedBiddingSignalsSlotSizeMode` fields.  Once a seller initiates an on-device auction on a publisher page, the browser checks each participating interest group for these fields, and makes an uncredentialed (cookieless) HTTP fetch to a URL of the form:
 
-    https://www.kv-server.example/getvalues?hostname=publisher.com&keys=key1,key2&interestGroupNames=name1,name2&experimentGroupId=12345
+    https://www.kv-server.example/getvalues?hostname=publisher.com&keys=key1,key2&interestGroupNames=name1,name2&experimentGroupId=12345&slot-size=100,200
 
-The base URL `https://www.kv-server.example/getvalues` comes from the interest group's `trustedBiddingSignalsURL`, the hostname of the top-level webpage where the ad will appear `publisher.com` is provided by the browser, `experimentGroupId` comes from `perBuyerExperimentGroupIds` if provided, `keys` is a list of `trustedBiddingSignalsKeys` strings, and `interestGroupNames` is a list of the names of the interest groups that data is being fetched for.  The requests may be coalesced (for efficiency) across any number of interest groups that share a `trustedBiddingSignalsURL` (which means they also share an owner).
+The base URL `https://www.kv-server.example/getvalues` comes from the interest group's `trustedBiddingSignalsURL`, the hostname of the top-level webpage where the ad will appear `publisher.com` is provided by the browser, `experimentGroupId` comes from `perBuyerExperimentGroupIds` if provided, `keys` is a list of `trustedBiddingSignalsKeys` strings, and `interestGroupNames` is a list of the names of the interest groups that data is being fetched for.  `trustedBiddingSignalsSlotSizeMode` is one of `none` (which is the default), `slot-size`, and `all-slots-requested-sizes`.  In the second case, `&slotSize=<width>,<height>` is appended to the URL, where width and height are the normalized width and height from the `requestedSize` of the (component) auction's AuctionConfig.  "Normalized" means units are always appended, and trailing zeros are removed, so {width: 62.50sw", height: "10.0" becomes "62.5sw,10px".  In the `all-slots-requested-sizes` case, `&slotSizes=<width1>,<height1>,<width2>,<height2>,...` is appended, where all sizes are taken from the (component) auction's `allSlotsRequestedSizes` value.  If the corresponding value is not present in the auction, no value is appended.  The requests may be coalesced (for efficiency) across any number of interest groups that share a `trustedBiddingSignalsURL` and `trustedBiddingSignalsSlotSizeMode` (which means they also share an owner).
```

Comment:
Disconnect between the illustrative URL

`&slot-size=100,200`

and the paragraph

>In the second case, `&slotSize=<width>,<height>` is appended to the URL,

Reply:
The first is a sample URL, rather than a template.  The second is telling you the what the values mean.  This is really the only place we explicitly mention that width is first (admittedly, the next line demonstrates it is), so I think this is probably the way to go?

Reply:
I was referring to the parameter name disconnect, which in the URL is supposed to have fidelity to what Chrome will fetch, yes?

Reply:
I completely missed that.  You're correct - and this is actually a bug in the current implementation. It should be using camelCase like everything else, but it is not.  Thanks so much for catching this before we shipped anything!  (Well, mostly - it's enabled on Canary with the wrong value).  The all-slots-requested-sizes string also doesn't match the implementation.  Anyhow, both issues should be fixed (Though still need to fix Chrome itself)

Reply:
You're welcome. I'll follow [the bug you filed](https://bugs.chromium.org/p/chromium/issues/detail?id=1512132).